DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -92,10 +95,19 @@ public string StartDateYear
             get => new SelectElement(YearDropDown).SelectedOption.Text;
             set => new SelectElement(YearDropDown).SelectByText(value);
         }
+
         public string StartDateDay
         {
             get => new SelectElement(DayDropDown).SelectedOption.Text;
             set => new SelectElement(DayDropDown).SelectByText(value);
         }
+
+        public void ClickOnSettingsLnk()
+        {
+            SettingsLnk.Click();
+        }
+
+        public string CurrenrCurrencyFld => CurrentCurrency.GetAttribute("textContent");
+
```

Comment:
1. Please remove extra line after.
2. Try `Text`
```suggestion
        public string CurrenrCurrencyFld => CurrentCurrency.Text;
```